src/screens/HomeScreen.js

```diff
@@ -133,32 +139,44 @@ const styles = StyleSheet.create({
     margin: moderateScale(10),
   },
   recordContainer: {
-    width: verticalScale(390),
-    height: scale(120),
+    width: moderateScale(360),
+    height: moderateScale(110),
+    marginLeft: moderateScale(7),
     marginBottom: moderateScale(10),
-    backgroundColor: colors.lightOrange,
+    backgroundColor: colors.grey,
+    borderRadius: moderateScale(10),
   },
   recordTextConteiner: {
     marginLeft: moderateScale(10),
     marginRight: moderateScale(10),
     marginTop: moderateScale(5),
+    flexDirection: "column",
+    justifyContent: "center",
+    flexGrow: 1,
+    width: 0,
   },
   row: {
     flexDirection: "row",
   },
   image: {
-    width: verticalScale(108),
-    height: scale(90),
+    width: scale(90),
+    height: moderateScale(90),
     marginLeft: moderateScale(6),
-    marginTop: scale(5),
+    marginTop: moderateScale(11),
+    marginBottom: moderateScale(5),
+    borderRadius: moderateScale(15),
+  },
+  icon: {
```

Comment:
It would be great if you can locate the arrow in the middle
Erase marginTop and add 
alignSelf: "center",
on the icon style